js/privatebin.js

```diff
@@ -408,7 +437,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
                 DOMPurify.sanitize(
                     element.html().replace(
                         /(((https?|ftp):\/\/[\w?!=&.\/-;#@~%+*-]+(?![\w\s?!&.\/;#~%"=-]>))|((magnet):[\w?=&.\/-;#@~%+*-]+))/ig,
-                        '<a href="$1" rel="nofollow noopener noreferrer">$1</a>'
+                        '<a href="$1" target="_blank" rel="nofollow noopener noreferrer">$1</a>'
```

Comment:
Interesting we did not use this here… Should have been used, since we actually want that.
I've searched some issues and this should be fixed since https://github.com/PrivateBin/PrivateBin/issues/61 already and https://github.com/PrivateBin/PrivateBin/issues/451.
As we use `noopener` here though, that should be fine.
(But please check DomPurify does not remove that or so, so it is properly sanitized.)